### ISSUE TITLE
[Windows] Most SWT related tests fail since I20240711-1800

### ISSUE THREAD

**Issue by iloveeclipse** [eclipse-platform/eclipse.platform.swt#1344]
See first build fails: https://download.eclipse.org/eclipse/downloads/drops4/I20240711-1800/testResults.php

Most recent build: https://download.eclipse.org/eclipse/downloads/drops4/I20240713-1800/testResults.php

**Comment by HeikoKlare** [comment]
Thank you, @iloveeclipse!
We first expected it to be related to some of the changes for Windows HiDPI support improvements, but we have not merged anything in the timeframe at which builds started to fail.

May also be related to the general, current issues with the infrastructure (https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/4805), which are usually solved by restarting the according machines. So let's hope it's fixed by the restart.

**Comment by HannesWell** [comment]
I can add another possible cause to the list:
https://github.com/eclipse-cbi/jiro/pull/379

The now longer path names might also cause problems due to the path length limitations active by default on Windows (btw. should we ask the it team to disable the limit in the registry?).
But since the first run after that change was applied and the testing I conducted in advance didn't show any error it is of course possible that there is just another memory leak and the mentioned change isn't the cause. 

And thanks Andrey for having an eye on th is topic!

**Comment by iloveeclipse** [comment]
Tha machine was restarted, but we were not able to connect to it on SDK build.

Now agent should have been connected manually, I've started new SDK build: https://ci.eclipse.org/releng/job/Builds/job/I-build-4.33/67/

**Comment by HeikoKlare** [comment]
Latest I-Build results look much better: https://download.eclipse.org/eclipse/downloads/drops4/I20240716-0250/testResults.php

**Comment by iloveeclipse** [comment]
@HeikoKlare : I see FindReplaceDialogTest failures on Windows: could you please follow up?

**Comment by HeikoKlare** [comment]
Yes, I am already preparing a tracking issue and will have a look.
